course_catalog/factories.py

```diff
@@ -341,3 +342,29 @@ def topics(self, create, extracted, **kwargs):
 
     class Meta:
         model = UserList
+
+
+class VideoResourceFactory(LearningResourceFactory):
+    """Factory for VideoResource"""
+
+    video_id = factory.Sequence(lambda n: "VIDEO-%03d.MIT" % n)
+    platform = FuzzyChoice([PlatformType.youtube.value])
+
+    full_description = factory.Faker("text")
+    image_src = factory.Faker("image_url")
+    image_description = factory.Faker("text", max_nb_chars=300)
```

Comment:
You need to remove image_description here as well

Reply:
Do'h, good catch, I'll fix that on the next PR for search. It's unused right now so it doesn't break anything at the moment